lib/pacemaker/pcmk_sched_native.c

```diff
@@ -1473,7 +1473,7 @@ allowed_nodes_as_list(pe_resource_t *rsc, pe_working_set_t *data_set)
         allowed_nodes = g_hash_table_get_values(rsc->allowed_nodes);
     }
 
-    if (pcmk_is_set(data_set->flags, pe_flag_stdout)) {
+    if (data_set->priv != NULL || pcmk_is_set(data_set->flags, pe_flag_stdout)) {
         allowed_nodes = g_list_sort(allowed_nodes, sort_node_uname);
```

Comment:
Since you set priv in the daemons, this will now always be done ... the goal was to avoid the overhead in the daemons

Reply:
I think the plan here was that I wanted to make sure things got logged, but that can be done without sorting.

Reply:
I've fixed this so the sorting no longer gets called, though I'm not thrilled about how I'm doing it.